main/src/ca/uwaterloo/flix/language/phase/UllmansAlgorithm.scala

```diff
@@ -116,7 +117,7 @@ object UllmansAlgorithm {
               changed = true
             }
 
-          case DependencyEdge.Strong(headSym, bodySym, edgeLoc) =>
+          case currentEdge@DependencyEdge.Strong(headSym, bodySym, _) =>
```

Comment:
We usually dont use `@` variables. You already have `edge` in scope here.

Reply:
A `DependencyEdge.Strong` type is needed and edge is just of DependencyEdge type

Reply:
OK